src/components/context-menus/ThemeListContextMenu.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
```

Comment:
Is this file being used anywhere?

Reply:
It was leftover from before. Fixed in acab commit